### ISSUE TITLE
Inline Chat Does Not Work In Accessibility Mode, Progress Reporting is Only Supported For Live Sessions

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#27707]
Testing -188537

![Code_-_Insiders_wVCfJlfUdT](https://github-com/microsoft/vscode/assets/2931520/f4c730c1-2c44-401b-947d-cd1f1514e4ce)
